bugbug/models/duplicate.py

```diff
@@ -147,4 +147,4 @@ def get_labels(self):
         return classes, [0, 1]
 
     def get_feature_names(self):
-        return self.extraction_pipeline.named_steps["union"].get_feature_names_out()
+        return self.extraction_pipeline.named_steps["union"].get_feature_names_out()
```

Comment:
Unrelated change. Similar to the unrelated change in `.gitignore`.

Reply:
You mean that I should add the empty line at the end of those files back?

Reply:
Yes